packages/api-key-stamper/src/utils.ts

```diff
@@ -1,5 +1,11 @@
 import { pointDecode } from "./tink/elliptic_curves";
-import { stringToBase64urlString } from "@turnkey/encoding";
+import {
+  stringToBase64urlString,
+  base64urlToBuffer,
+  uint8ArrayToHexString,
+} from "@turnkey/encoding";
+
+const DEFAULT_JWK_MEMBER_BYTE_LENGTH = 32;
```

Comment:
super tiny nit, if we plan on setting this in multiple files maybe we can export it from @turnkey/encoding